_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -259,14 +300,14 @@ A LocalAttribute contains the local metadata for an [Attribute](#attributes). Th
+The LocalAttributeShareInfo helps to keep track of how the LocalAttribute was received/sent, from whom it was received/who sent it, as well as which LocalAttribute it was copied from (in case of a shared IdentityAttribute). For example, this enables us to track back who we shared a certain IdentityAttribute with, so we are able to notify each of them when changing it. The Attribute can be either transmitted via Request or Notification. Hence, only one of `requestReference` and `notificationReference` can be set.
 
-| Name                  | Type                    | Description                                                                                                                         |
-| --------------------- | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
-| peer                  | `string`                | The Address of the Identity the LocalAttribute was received from/shared with.                                                       |
-| requestReference      | `string` \| `undefined` | The ID of the LocalRequest the LocalAttribute was received in/sent with. If this is set, `notificationReference` must be undefined. |
-| notificationReference | `string` \| `undefined` | The ID of the Notification the LocalAttribute was received in/sent with. If this is set, `requestReference` must be undefined.      |
-| sourceAttribute       | `string` \| `undefined` | If the LocalAttribute is a copy of a RepositoryAttribute, then this property contains the ID of the RepositoryAttribute.            |
+| Name                  | Type                    | Description                                                                                                                                               |
+| --------------------- | ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| peer                  | `string`                | The Address of the Identity the LocalAttribute was received from/shared with.                                                                             |
+| requestReference      | `string` \| `undefined` | The ID of the [LocalRequest](#localrequest) the LocalAttribute was received in/sent with. If this is set, `notificationReference` must be undefined.      |
+| notificationReference | `string` \| `undefined` | The ID of the [LocalNotification](#localnotification) the LocalAttribute was received in/sent with. If this is set, `requestReference` must be undefined. |
+| sourceAttribute       | `string` \| `undefined` | If the LocalAttribute is a copy of a RepositoryAttribute, then this property contains the `id` of the RepositoryAttribute.                                |
```

Comment:
I think it is a good idea to change the formatting from ID to `id` when refering to a property of a data object. I think it would be useful to insert a corresponding link then (in the case of "RepositoryAttribute", "LocalAttribute" must be used (the term "RepositoryAttribute" is defined there because its a certain kind of a "LocalAttribute")). 

For reasons of consistency, the formatting of the "ID of the LocalNotification" or the "ID of the LocalRequest" should be changed then to `id` too. Same for other locations.